scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
+    """Calculates degree of association between variables
+    that are nominal or greater.
+
+    Allows for specification of one of four related methods,
+    Tschuprow's T, Pearson's Contingency Coefficient, Cramer's V and Phi.
+
+    Parameters
+    ----------
+    arr : array-like
+        The array of observed values
+    chi2_stat : float
```

Comment:
Why must this be a mandatory input if it can simply be calculated from the observed values? This calculation is not performance-critical, so I think it's best to just remove this parameter, and calculate it within `association()`.

Reply:
I agree with @rgommers.  While I understand that it provides a potential optimization if the user has already computed the statistic, I'd rather simplify the API and have the function compute this internally.

Reply:
I made this change. I was a little unclear as to how to inherit arguments from chi2_contingency. If i should reference them explicitly in the function or use *args/ **kwargs and mention them in the doc string?